frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -198,12 +208,16 @@ function Application({ getWidget }: DashboardProps): JSX.Element {
 								onClickHandler={(event, element, chart, data): void => {
 									onClickHandler(event, element, chart, data, 'Rate');
 								}}
-								widget={getWidget([
-									{
-										query: `sum(rate(signoz_latency_count{service_name="${servicename}", operation=~\`${operationsRegex}\`${resourceAttributePromQLQuery}}[5m]))`,
-										legend: 'Operations',
-									},
-								])}
+								widget={getWidgetQueryBuilder({
```

Comment:
let's memo the response of this function to avoid multiple calculation

Reply:
@palashgdev  getWidgetQueryBuilder is again calling another function called operationPerSec. Did you mean we should memoize that function?

Reply:
it will be automatically asking to wrap in memo or callback